xain_fl/coordinator/metrics_store.py

```diff
@@ -0,0 +1,55 @@
+# pylint: disable=missing-docstring
+from abc import ABC, abstractmethod
+from datetime import datetime
+from typing import Dict
+
+from influxdb import InfluxDBClient
+from numpy import ndarray
+
+from xain_fl.config import MetricsConfig
+
+
+def current_time():
+    return datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
+
+
+class ABCMetricsStore(ABC): # pylint: disable=too-few-public-methods
+    """An abstract metric store."""
+
+    @abstractmethod
+    def write_metrics(self, participant_id: str, metrics: Dict[str, ndarray]) -> None:
+        """
+        Args:
+
+        participant_ids: The list of IDs of the participants selected
+            to participate in this round.
+        metrics :
+        """
+
+
+class NoMetricsStore(ABCMetricsStore): # pylint: disable=too-few-public-methods
```

Comment:
In `xain-sdk` I called the storage counterpart of this class `DummyStore`. Maybe we should agree on a naming convention for these classes that do nothing.